documentation/assemblies/managing/assembly-drain-cleaner.adoc

```diff
@@ -8,7 +8,8 @@
 [role="_abstract"]
 Kafka and ZooKeeper pods might be evicted during Kubernetes upgrades, maintenance or pod rescheduling.
 If your Kafka broker and ZooKeeper pods were deployed by Strimzi, you can use the Strimzi Drain Cleaner tool to handle the pod evictions.
-You need to set the `podDisruptionBudget` for your Kafka deployment to `0` (zero) for the Strimzi Drain Cleaner to work.
+Since the Strimzi Drain Cleaner will handle the eviction instead of Kubernetes, you need to set the `podDisruptionBudget` for your Kafka deployment to `0` (zero).
+Kubernetes will then no longer interfere.
```

Comment:
```suggestion
Kubernetes will then no longer be involved in the eviction process.
```

Reply:
Kubernetes will still try to evict, but receiving an error because of the PDB settings. For that reason I think "no longer be able to evict pods" is better.

Reply:
I think saying something like

```suggestion
Kubernetes will then no longer be allowed to evict the pod on its own.
```

might be better.